Rectangle/Defaults.swift

```diff
@@ -68,6 +68,8 @@ class Defaults {
     static let shortEdgeSnapAreaSize = FloatDefault(key: "shortEdgeSnapAreaSize", defaultValue: 145)
     static let cascadeAllDeltaSize = FloatDefault(key: "cascadeAllDeltaSize", defaultValue: 30)
     static let sixthsSnapArea = OptionalBoolDefault(key: "sixthsSnapArea")
+    static let quarterColumnsSnapAera = OptionalBoolDefault(key: "quarterColumnsSnapAera")
+    static let eightsSnapArea = OptionalBoolDefault(key: "eightsSnapArea")
```

Comment:
Keeping with convention, should be `eighths`. It looks like only bottom eighths was implemented, in which case a more applicable name would be `bottomEighthsSnapArea`? Since the sixthsSnapArea also handles the top, it feels a little incomplete to only have the bottom eighths implemented.